info/v1/machine.go

```diff
@@ -41,6 +41,7 @@ type Node struct {
 	Memory uint64  `json:"memory"`
 	Cores  []Core  `json:"cores"`
 	Caches []Cache `json:"caches"`
+	Pcis   []string   `json:"pcis"`
```

Comment:
What is this field supposed to contain? List of PCI IDs of devices? 
Also add to v2 API as well.

Reply:
Yes. This field contains a list of PCI IDs.
Such as: [ "0000:00:00.0", "0000:00:00.1", "0000:00:01.0"].
You can see the test cases in "machine/topology_test.go" as reference.

The field of 'Pcis' was belong to struct "Topology".
The struct of 'Topology' was already added into v2 API (.info/v2/machine.go).
I think this field can be obtained by accessing the topology in v2 API.
So 'Pcis' should not be added inot v2 API.
I don't know if my understanding is wrong.
Thanks.